api-outbound.yaml

```diff
@@ -144,6 +150,64 @@ paths:
         basicauth:
           username: "${NEXPOSE_API_USERNAME}"
           password: "${NEXPOSE_API_PASSWORD}"
+  /publish:
+    post:
+      description: Publish a completed scan event to an HTTP queue.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ScanNotification'
+      responses:
+        "200":
+          description: "Success"
+        "400":
+          description: "Invalid input"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-transportd:
+        backend: queue
+        enabled:
+          - "accesslog"
+          - "metrics"
+          - "requestvalidation"
+          - "timeout"
+          - "retry"
+        metrics:
+          pathtag: "queue"
```

Comment:
task: this is asking for the name of the tag not the value. this should be something like `client_path`. Alternatively, you can remove this value and `backendtag` below since both have default values defiend by the proxy.

Reply:
Ah, good to know. I've removed them.